gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/application/use_case/ImportApplicationCRDUseCase.java

```diff
@@ -150,6 +179,76 @@ private void createOrUpdateApplicationMetadata(Input input, BaseApplicationEntit
         }
     }
 
+    private void syncApplicationMembers(Input input, BaseApplicationEntity application) {
+        List<String> existingApplicationMembers = memberQueryService
+            .getMembersByReference(MembershipReferenceType.APPLICATION, application.getId())
+            .stream()
+            .filter(member -> {
+                for (Member.Role role : member.getRoles()) {
+                    if (role.isPrimaryOwner()) {
+                        return false;
+                    }
+                }
+                return true;
+            })
+            .map(Member::getId)
+            .collect(Collectors.toList());
+
+        if (!CollectionUtils.isEmpty(input.crd.getMembers())) {
+            input.crd
+                .getMembers()
+                .forEach(crdMember -> {
+                    if (PRIMARY_OWNER.name().equals(crdMember.getRole())) {
+                        throw new SinglePrimaryOwnerException(APPLICATION);
+                    }
+
+                    applicationCrudService.findById(application.getId(), input.auditInfo.environmentId());
+                    Optional<BaseUserEntity> optionalBaseUser = userDomainService.findBySource(
+                        input.auditInfo.organizationId(),
+                        crdMember.getSource(),
+                        crdMember.getSourceId()
+                    );
+                    if (optionalBaseUser.isEmpty()) {
+                        log.warn(
+                            "member with source [{}] and source id [{}] doesn't exist. It will not be imported ...",
+                            crdMember.getSource(),
+                            crdMember.getSourceId()
+                        );
+                        return;
+                    }
```

Comment:
I think we could almost skip that part as it will be sanitized in an upcoming PR